src/test/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/decisionmaker/actions/ModifyCacheMaxSizeActionTest.java

```diff
@@ -74,23 +74,46 @@ public void setUp() {
   public void testIncreaseCapacity() {
     NodeKey node1 =
         new NodeKey(new InstanceDetails.Id("node-1"), new InstanceDetails.Ip("1.2.3.4"));
-    ModifyCacheMaxSizeAction modifyCacheSizeAction =
-        new ModifyCacheMaxSizeAction(
+    ModifyCacheMaxSizeAction.Builder builder =
```

Comment:
This class is missing 2 important tests. When the Builder is unable to determine currentCacheMaxSizeInBytes or heapMaxSizeInBytes, you need to verify that build() returns a non-actionable action.

Reply:
Added tests around this